portfolio/src/main/webapp/script.js

```diff
@@ -27,3 +27,32 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+// Written with help from https://www.w3schools.com/howto/howto_html_include.asp
+function createNavigation() {
+    var total, i, element, file, xhttp;
+    // Loop through a collection of all HTML elements. 
+    total = document.getElementsByTagName("*");
+    for (i = 0; i < total.length; i++) {
+        element = total[i];
```

Comment:
If you only need the values in the array, you can simply write:

for (const element of total) {
  // handle element....
}

See the difference between for... of and for... in:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in